vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc

```diff
@@ -68,15 +69,13 @@ std::string HeadersToString(
 }
 
 std::string UrlResponseToString(
-    const std::string& url,
-    const int response_status_code,
-    const std::string& response,
-    const std::map<std::string, std::string>& headers) {
-  const std::string formatted_headers = HeadersToString(headers);
+    const UrlResponse& url_response) {
+  const std::string formatted_headers = HeadersToString(url_response.headers);
 
   return base::StringPrintf("URL Response:\n  URL: %s\n  Response "
-      "Status Code: %d\n  Response: %s\n  Headers:\n%s", url.c_str(),
-          response_status_code, response.c_str(), formatted_headers.c_str());
+      "Status Code: %d\n  Body: %s\n  Headers:\n%s",
+          url_response.url.c_str(), url_response.status_code,
+             url_response.body.c_str(), formatted_headers.c_str());
```

Comment:
indentation is wrong, needs 1 more space

Reply:
done